### PR TITLE
Update `MWC 4K 2020` / `MWC 4K 2021` with podium info

### PR DESCRIPTION
**Pull request by LeoFLT**
Fixes a wrong alt text on `MWC 4K 2020` and updates `MWC 4K 2021` with podium info.

### CODE REVIEW

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 1]
```diff
@@ -245,9 +238,18 @@ The osu!mania 4K World Cup 2021 is run by the [osu! team](/wiki/People/The_Team)
 
 ## Match results
 
-### Finals
+### Finals week 2
+
+Detailed statistics for Finals week 1 and 2 can be found [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vT2RgCDlCseyg6CNGnrGtnIRTadNqYt1wuZKkthLV9o4hsGl4gDwPiq2oAF1lp1JT0UougO8Z28Fvex/pubhtml "Google Sheets").
```
Comment (TPGPL) [inline wiki/Tournaments/MWC/2021_4K/en.md:243]:
```suggestion
Detailed statistics for Finals week 1 and 2 can be found [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vT2RgCDlCseyg6CNGnrGtnIRTadNqYt1wuZKkthLV9o4hsGl4gDwPiq2oAF1lp1JT0UougO8Z28Fvex/pubhtml).
```

**Comment by Bazzadwarf** [comment]
Also need to update the osu!mania 4K World Cup 2021 podium on the tournaments main page

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 2]
```diff
@@ -10,33 +10,35 @@ tags:
 
 ![](/wiki/shared/news/banners/mwc4k-2021.jpg)
```
Comment (TPGPL) [inline wiki/Tournaments/MWC/2021_4K/en.md:11]:
```suggestion
![MWC 4K 2021 banner](/wiki/shared/news/banners/mwc4k-2021.jpg)
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 3]
```diff
@@ -98,24 +100,15 @@ The osu!mania 4K World Cup 2021 is run by the [osu! team](/wiki/People/The_Team)
...
-¹ Losers bracket match\
-² Final matchup depends on the results of ¹
+![](img/podium.jpg "MWC 4K 2021 podium")
 
 ## Mappools
 
 ### Finals
 
-*Note: this mappool is used for both the Finals and Grand Finals.*
+*Note: this mappool was used for both Finals week 1 and 2.*
```
Comment (TPGPL) [inline wiki/Tournaments/MWC/2021_4K/en.md:111]:
```suggestion
*Note: This mappool was used for both Finals week 1 and 2.*
```

**Comment by LeoFLT** [comment]
Everything is fixed *I think*? Should be good to merge (although I probably need to revisit all the past articles and fix them as well at some point, they're riddled with small errors here and there)

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**